src/appleseed.studio/mainwindow/rendering/renderingmanager.h

```diff
@@ -162,15 +161,16 @@ class RenderingManager
 
   private:
     StatusBar&                                  m_status_bar;
-    QtRendererController                        m_renderer_controller;
+    QtRendererController                        m_renderer_controller; 
```

Comment:
Revert.

Reply:
I don't know why this is happening. It seems perfect on my editor.